test/libsolidity/smtCheckerTests/loops/do_while_bmc_iterations_4.sol

```diff
@@ -0,0 +1,15 @@
+contract C {
+	function f() public pure {
+		uint x;
+		do {
+			++x;
+		} while (true);
+		assert(x == 1);
+	}
+}
+// ====
+// SMTEngine: bmc
+// SMTSolvers: z3
+// BMCLoopIterations: 1
+// ----
+// Info 6002: BMC: 2 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

Comment:
Interesting, this should be "safe" because it's unrecheable, right? Can you add a copy of this test with BMCLoopIterations = 5, which should give the same result?

Reply:
Ah I see you have some tests like that already. It's fine then!